config/overlays/helm/manager.yaml

```diff
@@ -34,6 +34,8 @@ spec:
             value: "{{ .Values.sidecarConfiguration.envVarPrefix }}"
           - name: SIDECAR_SYNC_PROVIDER
             value: "{{ .Values.sidecarConfiguration.defaultSyncProvider }}"
+            - name: SIDECAR_EVALUATOR
+            value: "{{ .Values.sidecarConfiguration.evaluator }}"
```

Comment:
There's an indentation error here that is breaking helm packaging.

Reply:
resolved